superbench/benchmarks/__init__.py

```diff
@@ -16,7 +16,7 @@
         map(
             importlib.import_module, [
                 'superbench.benchmarks.{}'.format(module)
-                for module in ['model_benchmarks', 'micro_benchmarks', 'docker_benchmarks']
+                for module in ['model_benchmarks', 'micro_benchmarks', 'docker_benchmarks', 'directx_benchmarks']
```

Comment:
I think it's different with this. model, micro, and docker benchmark should be orthogonal with DirectX benchmark. DirectX should be similar as CPU, CUDA and ROCm

Reply:
do you think the source code of all DirectX benchmarks should be under microbenchmarks directly, or microbenchmarks/directxbenchmark?